.gitignore

```diff
@@ -52,22 +52,22 @@ dist/
 compile_commands.json
 
 # Example resources
-examples/test_data/open3d_downloads
+data/common/downloads
 examples/python/reconstruction_system/dataset/
-examples/test_data/fountain_small
-examples/test_data/Armadillo.ply
-examples/test_data/Bunny.ply
-examples/test_data/eagle.ply
-examples/test_data/voxelized.ply
+data/common/fountain_small
+data/common/Armadillo.ply
```

Comment:
(Future work) These are downloaded datasets. Let's not change them in this PR to keep things simple. In the next PR, this shall be moved to the downloads folder and we no longer need to git ignore them.